app/src/main/java/org/openimis/imispolicies/domain/PolicyRenewal.java

```diff
@@ -0,0 +1,178 @@
+package org.openimis.imispolicies.domain;
+
+import android.os.Parcel;
+import android.os.Parcelable;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+
+import java.util.Date;
+
+public class PolicyRenewal implements Parcelable {
+
+    private final int id;
+    @NonNull
+    private final String uuid;
```

Comment:
Yes, `UUID` is available in Android. I didn't use it because it changes how the string is represented. In the legacy code, the UUIDs are stored as received, with capital letters. In the Java implementation, the letters are lowercase. Therefore, it clashes when doing SQL requests as the strings are not the same anymore.
Also, I didn't see any benefits of enforcing the value on the client side. It would crash but without adding any benefits to the user.

Reply:
LGTM especially considering the app context.